packages/rocketchat-livechat/app/client/views/livechatWindow.html

```diff
@@ -47,7 +47,6 @@ <h1>{{title}}</h1>
 						{{> offlineForm (offlineData)}}
 					</div>
 				{{/if}}
-				{{> poweredBy }}
```

Comment:
@HenrikScholl can you please explain why you moved the `poweredBy` from here to the multiple other places?

Reply:
I changed that, because I set the fix position to the Input-Form.
In the Offline-Version is no Input-Form, so the "Powered by"-Logo  has the wrong Postion